lib/cosmos/dart/script/dart_decom_client.rb

```diff
@@ -0,0 +1,31 @@
+ENV['COSMOS_USERPATH'] = 'C:/git/COSMOS/demo'
+require 'cosmos'
+require 'cosmos/io/json_drb_object'
+Cosmos::Logger.level = Cosmos::Logger::DEBUG
+
+start_time = Time.utc(1970, 1, 1, 0, 0, 0)
+end_time = Time.utc(2020, 1, 1, 0, 0, 0)
```

Comment:
Use Time.now.utc instead of fixed time (that is depressingly close?)

Reply:
This is just an example script and meant to be hacked on, but that would be a good improvement.